### ISSUE TITLE
No matching version found for react-native-template-typescript@6.6.5

### ISSUE THREAD

**Issue by CyxouD** [react-native-community/react-native-template-typescript#235]
## Expected results

react-native-template-typescript@6.6.5 to be installed

## Observed results

react-native-template-typescript@6.6.5 is not installed

### Logs

Short log:
```
✖ Downloading template
error Error: Command failed: npm install --save --save-exact react-native-template-typescript@6.6.5
npm ERR! code ETARGET
npm ERR! notarget No matching version found for react-native-template-typescript@6.6.5.
npm ERR! notarget In most cases you or one of your dependencies are requesting
npm ERR! notarget a package version that doesn't exist.
```

Complete log:

```
0 info it worked if it ends with ok
1 verbose cli [
1 verbose cli   '/usr/local/Cellar/node/17.0.1/bin/node',
1 verbose cli   '/usr/local/bin/npm',
1 verbose cli   'install',
1 verbose cli   '--save',
1 verbose cli   '--save-exact',
1 verbose cli   'react-native-template-typescript@6.6.5'
1 verbose cli ]
2 info using npm@6.14.8
3 info using node@v17.0.1
4 verbose npm-session b2acce94c0406bae
5 silly install loadCurrentTree
6 silly install readLocalPackageData
7 http fetch GET 200 https://registry.npmjs.org/react-native-template-typescript 13ms (from cache)
8 silly registry:manifest no matching version for react-native-template-typescript@6.6.5 in the cache. Forcing revalidation.
9 http fetch GET 200 https://registry.npmjs.org/react-native-template-typescript 1024ms
10 silly fetchPackageMetaData error for react-native-template-typescript@6.6.5 No matching version found for react-native-template-typescript@6.6.5.
11 timing stage:rollbackFailedOptional Completed in 0ms
12 timing stage:runTopLevelLifecycles Completed in 1070ms
13 verbose type version
14 verbose stack react-native-template-typescript: No matching version found for react-native-template-typescript@6.6.5.
14 verbose stack     at pickManifest (/usr/local/lib/node_modules/npm/node_modules/npm-pick-manifest/index.js:122:13)
14 verbose stack     at /usr/local/lib/node_modules/npm/node_modules/pacote/lib/fetchers/registry/manifest.js:43:18
14 verbose stack     at tryCatcher (/usr/local/lib/node_modules/npm/node_modules/bluebird/js/release/util.js:16:23)
14 verbose stack     at Promise._settlePromiseFromHandler (/usr/local/lib/node_modules/npm/node_modules/bluebird/js/release/promise.js:517:31)
14 verbose stack     at Promise._settlePromise (/usr/local/lib/node_modules/npm/node_modules/bluebird/js/release/promise.js:574:18)
14 verbose stack     at Promise._settlePromise0 (/usr/local/lib/node_modules/npm/node_modules/bluebird/js/release/promise.js:619:10)
14 verbose stack     at Promise._settlePromises (/usr/local/lib/node_modules/npm/node_modules/bluebird/js/release/promise.js:699:18)
14 verbose stack     at _drainQueueStep (/usr/local/lib/node_modules/npm/node_modules/bluebird/js/release/async.js:138:12)
14 verbose stack     at _drainQueue (/usr/local/lib/node_modules/npm/node_modules/bluebird/js/release/async.js:131:9)
14 verbose stack     at Async._drainQueues (/usr/local/lib/node_modules/npm/node_modules/bluebird/js/release/async.js:147:5)
14 verbose stack     at Immediate.Async.drainQueues [as _onImmediate] (/usr/local/lib/node_modules/npm/node_modules/bluebird/js/release/async.js:17:14)
14 verbose stack     at processImmediate (node:internal/timers:464:21)
15 verbose cwd /private/var/folders/9p/19tvy00s3x76sgxkkch_1h980000gn/T/rncli-init-template-oCPObH
16 verbose Darwin 20.6.0
17 verbose argv "/usr/local/Cellar/node/17.0.1/bin/node" "/usr/local/bin/npm" "install" "--save" "--save-exact" "react-native-template-typescript@6.6.5"
18 verbose node v17.0.1
19 verbose npm  v6.14.8
20 error code ETARGET
21 error notarget No matching version found for react-native-template-typescript@6.6.5.
22 error notarget In most cases you or one of your dependencies are requesting
22 error notarget a package version that doesn't exist.
23 verbose exit [ 1, true ]
```

## Steps to reproduce

1. npx react-native init WordFromWord --template react-native-template-typescript@6.6.5

## Important

6.6.4 version works correctly: "npx react-native init WordFromWord --template react-native-template-typescript@6.6.4"

**Comment by emin93** [comment]
Good catch, something prevented the release of this version to NPM. Since it's too long ago (Travis was used), I can't see what it was.

In any case, I have now manually released `6.6.5`. Can you try again?

**Comment by CyxouD** [comment]
@emin93 Thank you, it works correctly now!
